climpred/metrics.py

```diff
@@ -147,9 +89,26 @@ def get_metric_function(metric):
             metric = _nmae
         elif metric.lower() == 'uacc':
             metric = _uacc
+        elif metric.lower() == 'msss_murphy':
+            metric = _msss_murphy
+        elif metric.lower() in ['c_b', 'conditional_bias']:
+            metric = _conditional_bias
+        elif metric.lower() in ['u_b', 'unconditional_bias', 'bias']:
+            metric = _bias
+        elif metric.lower() == 'std_ratio':
+            metric = _std_ratio
+        elif metric.lower() == 'bias_slope':
+            metric = _bias_slope
+        elif metric.lower() == 'less':
+            metric = _less
+        elif metric.lower() == 'crps':
+            metric = _crps
+        elif metric.lower() == 'crpss':
+            metric = _crpss
         else:
             raise ValueError("""Please supply a metric from the following list:
                 'pearson_r'
+                'pearson_r_p_value'
```

Comment:
If you proceed to use the dict method, instead of typing all of these out, you can just type, `list(dict.keys())`